app/assets/javascripts/application/open_all.js.coffee

```diff
@@ -0,0 +1,7 @@
+jQuery ->
+
+$("#open_all").click ->
+
+  imports = $(this).data('bgeigie');
+  for dataset in imports
+    window.open("bgeigie_imports/"+dataset.id) unless dataset.approved
```

Comment:
New line would be good here (note the 🚫 shown on github)

Reply:
Most of editors have options or plugins to insert new line at the end of file.